src/calculateRentalCost.js

```diff
@@ -2,9 +2,30 @@
  * @param {number} days
  *
  * @return {number}
+ *
+ *
+ *
  */
+
+const PRICE = 40;
+
+const DAYS_SMALL_DISCOUNT = 3;
+const SMALL_DISCOUNT = 20;
+
+const DAYZ_BIG_DISCOUNT = 7;
+const BIG_DISCOUNT = 50;
```

Comment:
You don't use those consts in other function, but they make program more slow, because exist all time. Rewrite this in function